airflow/providers/github/CHANGELOG.rst

```diff
@@ -20,6 +20,14 @@
 Changelog
 ---------
 
+2.2.2
+.....
+
+Bug Fixes
+~~~~~~~~~
+
+* ``Allow for 'tag_name' in GithubTagSensor to be templated (#29842)``
+
```

Comment:
We are adding changes to change log only in breaking changes (as in that case the commit message is not enough to explain what action users should take) since now the change is backward compatible you can remove this